include/api/CDataFrameBoostedTreeRunner.h

```diff
@@ -61,6 +64,9 @@ class API_EXPORT CDataFrameBoostedTreeRunner final : public CDataFrameAnalysisRu
     TBoostedTreeFactoryUPtr m_BoostedTreeFactory;
     TBoostedTreeUPtr m_BoostedTree;
     std::atomic<std::int64_t> m_Memory;
+
+    bool restoreBoostedTree(core::CDataFrame& frame,
+                            CDataFrameAnalysisSpecification::TDataSearcherUPtr& restoreSearcher);
```

Comment:
nit: as per coding standards, can we keep this in the private member functions section.

Reply:
Done. Unfortunately, CLion misplaces the extracted methods sometimes.